Releases.md

```diff
@@ -6,6 +6,21 @@ https://github.com/denoland/deno/releases
 We also have one-line install commands at
 https://github.com/denoland/deno_install
 
+### 1.0.2 / 2020.05.22
+
+Changes in the CLI:
+
+fix: --inspect flag working like --inspect-brk (#5697) fix: disallow http
+imports for modules loaded over https (#5680) fix: redirects handling in module
+analysis (#5726) fix: SWC lexer settings and silent errors (#5752) fix: TS type
+imports (#5733) fix(fmt): Do not panic on new expr with no parens. (#5734)
+fix(cli/js/streams): high water mark validation (#5681)
+
+Changes in std version 0.53.0:
+
+fix(std/http): file_server's target directory (#5695) feat(std/hash): add md5
```

Comment:
You have some formatting errors here. 

Here's a trick I use: `git log v1.0.1... --format="- %s"`